docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst

```diff
@@ -318,43 +320,38 @@ As most of ROS 2 CLI Introspection is executed by adding a node into the network
 and some create their own nodes), using Discovery Server v2 we will find that most of these functionalities are
 limited and do not have all the information.
 
-The Discovery Server v2 functionality allows every node running as a **Server** (a kind of *Participant type*)
-to know and share all the participants and topics information with every other server matched.
-In this sense, a server can be configured alongside ROS 2 introspection, since then the introspection tool will
-be able to discover every entity in the network that is using the Discovery Server protocol.
+The Discovery Server v2 functionality allows every node running as a |SUPER_CLIENT|, a kind of *Participant type* that
+connects to a |SERVER| and receives all the discovery information from it (instead of just what it needs).
+In this sense, ROS 2 introspection tools can be configured as *super client*, thus being able to discover every entity
+that is using the Discovery Server protocol within the network.
 
 
 Daemon's related commands
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The ROS 2 Daemon is used in several ROS 2 CLI introspection commands. It adds a ROS 2 Node to the network in order
 to receive all the data sent.
-This section will explain how to use ROS 2 CLI with ROS 2 Daemon running as a **Server**.
+This section will explain how to use ROS 2 CLI with ROS 2 Daemon running as a **Super Client**.
```

Comment:
Maybe a sentence stating that for the ROS 2 daemon to fulfill its purpose when using the Discovery Server mechanism must be configured as `Super Client`, could clarify this paragraph.